### DISCUSSION TITLE
Add News Letter option

### DISCUSSION THREAD

**Discussion by mukhtharcm** [adityatelange/hugo-PaperMod#381] [Ideas]
It would have been cool if we could add a newsletter to our site just like we add google analytics now.
I tried using tinyletter by editing templates, but its design was very much annoying.
Is there any proper way we can add newsletter services?

**Comment by adityatelange** [comment]
Google analytics template is a Hugo feature and not PaperMod specific.

Adding newsletter services can be done by using scripts provided by them and adding them to extended_head or extended_footer.
Also you need to tweak out the templates and/or shortcodes to add matching styles if you want to.
